### ISSUE TITLE
Readme more info 

### ISSUE THREAD

**Issue by nicolidin** [HugoKempfer/Vim_Dotfiles#2]
Can you please add information about installing vim plug.
I always take time to understand why can't I do PlugInstall.
It is just cause we have to download vim-plug before: 

command line to dl vim-plug: `sh -c 'curl -fLo "${XDG_DATA_HOME:-$HOME/.local/share}"/nvim/site/autoload/plug.vim --create-dirs \
       https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim'`

**Comment by HugoKempfer** [comment]
Yeah, you're right! VimPlug is supposed to be installed during the first startup-time but it often fails..
I'll update the documentation. Also, your command is right, but only working for Linux + Neovim. The best is just to link to the `VimPlug` official doc.
